app/views/Export/ExportSelectHA.js

```diff
@@ -0,0 +1,164 @@
+                icon={Icons.Close}
+                size={22}
+                onPress={() => navigation.navigate('SettingsScreen')}
+              />
+            </View>
+            <Typography use='headline2' style={styles.exportSectionTitles}>
+              {t('export.select_ha_title')}
+            </Typography>
+          </View>
+          <Separator />
+          <FlatList
+            keyExtractor={({ name }, i) => `${name}:${i}`}
+            renderItem={({ item: HA }) => (
+              <TouchableHighlight
+                underlayColor={'#E5E7FA'}
+                style={{
+                  paddingVertical: 20,
+                  paddingHorizontal: 24,
+                }}
+                onPress={() => toggleSelected(HA)}>
+                <View
+                  style={{
+                    justifyContent: 'space-between',
+                    flexDirection: 'row',
+                  }}>
+                  <Typography
+                    style={{ fontWeight: '500', paddingRight: 30 }}
+                    use='body1'>
+                    {HA.name}
+                  </Typography>
+                  {/* Preserve icon positioning to prevent adding wrapping  */}
+                  <View style={{ opacity: selectedAuthority === HA ? 1 : 0 }}>
+                    <SvgXml xml={Icons.Checkmark} width={24} height={24} />
+                  </View>
+                </View>
+              </TouchableHighlight>
+            )}
+            alwaysBounceVertical={!!authorities}
+            style={{ flex: 1 }}
+            contentContainerStyle={{ flexGrow: 1 }}
+            data={authorities}
+            ListEmptyComponent={
+              <View style={{ flex: 1, justifyContent: 'center' }}>
+                <ActivityIndicator size={'large'} />
+              </View>
+            }
+            ItemSeparatorComponent={() => <Separator />}
+          />
+        </SafeAreaView>
+        <View style={styles.card}>
+          <SafeAreaView style={{ paddingBottom: 44 }}>
+            <View style={{ paddingHorizontal: 24 }}>
+              <Button
+                style={styles.exportButton}
+                label={t('common.next')}
+                onPress={() =>
+                  navigation.navigate('ExportCodeInput', { selectedAuthority })
+                }
+                disabled={!selectedAuthority}
+              />
+            </View>
+          </SafeAreaView>
+        </View>
+      </View>
+    </Theme>
+  );
+};
+
+const Separator = () => (
+  <View
+    style={{
+      backgroundColor: Colors.DIVIDER,
+      height: StyleSheet.hairlineWidth,
+      width: '100%',
+    }}
+  />
+);
+
+const styles = StyleSheet.create({
+  wrapper: { flex: 1, backgroundColor: Colors.INTRO_WHITE_BG },
+  exportSectionTitles: {
+    fontWeight: '500',
+    color: Colors.VIOLET_TEXT_DARK,
+    fontFamily: fontFamily.primaryMedium,
+  },
+  card: {
+    backgroundColor: Colors.WHITE,
+    shadowColor: Colors.BLACK,
+    shadowOffset: {
```

Comment:
This is a pretty arbitrary shadow. 
I like using https://ethercreative.github.io/react-native-shadow-generator/